Assets/MixedRealityToolkit/Utilities/Editor/FileUtilities.cs

```diff
@@ -13,14 +14,35 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
     public static class FileUtilities
     {
         /// <summary>
-        /// Locates the files that match the specified name within the Assets folder (application data path) structure.
+        /// Locates the files that match the specified name within the Assets folder structure.
         /// </summary>
         /// <param name="fileName">The name of the file to locate (ex: "TestFile.asmdef")</param>
+        /// <param name="searchFullFileSystem">If true, searches the full application data path. If false, only searches Unity's known asset database.</param>
         /// <returns>Array of FileInfo objects representing the located file(s).</returns>
-        public static FileInfo[] FindFilesInAssets(string fileName)
+        public static FileInfo[] FindFilesInAssets(string fileName, bool searchFullFileSystem = false)
```

Comment:
Nit: Consider maybe searchAllApplicationData instead of searchFullFileSystem (because full file system.. is that like, root of the C: drive? Is it the "/" folder?